docs/git-compatibility.md

```diff
@@ -9,6 +9,37 @@ See `jj help git` for help about the `jj git` family of commands, and e.g.
 briefer help).
 
 
+## Supported features
+
+* Configuration: No, the only configuration from Git (e.g. in `~/.gitconfig`)
+  that's respected is the configuration of remotes. Feel free to file a bug
+  if you miss any particular configuration options.
+* Authentication: Partial. Only `ssh-agent` or a password-less key file at
+  `~/.ssh/id_rsa` (and only at exactly that path).
+* Branches: Yes. You can read more about
+  [how branches work in Jujutsu](branches.md)
+  and [how they interoperate with Git](#branches).
+* Tags: Partial. You can check out tagged commits by name, but you cannot
```

Comment:
Tag objects or lightweight tags? For lightweight tags, if supported, can you modify/delete them?

Reply:
> Tag objects or lightweight tags? For lightweight tags, if supported, can you modify/delete them?

Both. Updated the text.